app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1 @@
+$facebook_blue: #4267b2;
```

Comment:
there's already a variables file and it contains $facebook-blue. Can you look at that file before adding new variables?

Reply:
where is that variable located?

Reply:
stylesheets/helpers/_variables.scss

Reply:
nvm, it should be gone now.